dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/add-fees-to-an-existing-payment-group/add-fees-to-an-existing-payment-group.event-handler.test.ts

```diff
@@ -0,0 +1,202 @@
+import { EntityManager } from 'typeorm';
+import {
+  UtilisationReportEntityMockBuilder,
+  DbRequestSource,
+  FeeRecordEntityMockBuilder,
+  UtilisationReportEntity,
+  PaymentEntityMockBuilder,
+  FeeRecordEntity,
+  PaymentEntity,
+} from '@ukef/dtfs2-common';
+import { handleUtilisationReportAddFeesToAnExistingPaymentGroupEvent } from './add-fees-to-an-existing-payment-group.event-handler';
+import { FeeRecordStateMachine } from '../../../fee-record/fee-record.state-machine';
+import { feeRecordsMatchAttachedPayments } from '../helpers';
+
+jest.mock('../helpers');
+
+describe('handleUtilisationReportAddFeesToAnExistingPaymentGroupEvent', () => {
+  const tfmUserId = 'abc123';
+  const requestSource: DbRequestSource = {
+    platform: 'TFM',
+    userId: tfmUserId,
+  };
+
+  const mockSave = jest.fn();
+  const mockEntityManager = {
+    save: mockSave,
+  } as unknown as EntityManager;
+
+  const aReconciliationInProgressReport = () => UtilisationReportEntityMockBuilder.forStatus('RECONCILIATION_IN_PROGRESS').build();
+
+  const aFeeRecordForReport = (report: UtilisationReportEntity, id: number) => FeeRecordEntityMockBuilder.forReport(report).withId(id).build();
```

Comment:
I think this line is short enough that we should just inline calls to this, especially as this is what the entity mock builders are designed for.